modules/qualify/qualify.c

```diff
@@ -241,11 +241,13 @@ static void call_stop_qualify(struct call *call, bool closed)
 		return;
 
 	qualle = le->data;
+	tmr_cancel(&qualle->to_tmr);
+	tmr_cancel(&qualle->int_tmr);
 
-	if (closed)
+	if (closed) {
```

Comment:
This function has two different purposes.

I suggest to remove `closed` parameter and move the branch to the switch under `UA_EVENT_CALL_CLOSED`

Reply:
Refactored.